src/modules/chat/ChatModuleNew.tsx

```diff
@@ -46,11 +54,44 @@ export const ChatModuleNew = () => {
   },[connectedUser]);
 
   return (
-    <>
-      <ChatLocalContextProviderNew>
-        <ChatSidebarSectionNew />  
-        <ChatBoxSectionNew />
-      </ChatLocalContextProviderNew>
-    </>
+    <Container>
+      {isLoading ? 
```

Comment:
this condition is wrong. it should be !isLoading

Reply:
corrected and now it is working